### PR TITLE
Feature2

### PR DESCRIPTION
**Pull request by iamasianman**
love to hear your guys feed back, I had to fix a bug of my GET request and fixed some typos. here is some backend stuff that I finally got to have a successfully get request. As for the front end I am just going to hard code it so I get the basic layout of my page and im going to refactor it by adding real dummy data by this week.

### CODE REVIEW

**Comment by iamasianman** [comment]
@blbst123 @kimbyongjin @hammertube sorry i forgot to tag you guys eariler about my pull request but here is part 1.

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (blbst123) [inline .babelrc]: Good

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 2
```
Comment (blbst123) [inline .circleci/config.yml]: Good

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,25 @@
+## MAC files ##
```
Comment (blbst123) [inline .gitignore]: Good

File: README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# FEC PROJECT
```
Comment (blbst123) [inline README.md]: What's this?

File: client/dist/index.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Comments</title>
+    <link rel="stylesheet" href="styles.css" />
```
Comment (blbst123) [inline client/dist/index.html]: Refactor to use css modules

File: client/src/components/Comments.test.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import Enzyme from "enzyme";
```
Comment (blbst123) [inline client/src/components/Comments.test.js]: Good start, make sure to add more useful tests

Comment (blbst123) [inline client/src/components/Comments.test.js]: This should be a separate PR.

File: package.json [hunk 7]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "react-dev": "webpack -d --watch",
     "server-dev": "nodemon server/index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --watch"
```
Comment (blbst123) [inline package.json]: Do you need to be watching your tests?

File: package.json [hunk 8]
```diff
@@ -20,15 +20,24 @@
...
+    "jest": "^24.8.0",
+    "mongoose": "^5.5.10",
     "webpack": "^4.31.0",
     "webpack-cli": "^3.3.2"
   },
   "dependencies": {
+    "axios": "^0.18.0",
     "express": "^4.16.4",
     "react": "^16.8.6",
+    "react-com": "^1.0.6",
```
Comment (blbst123) [inline package.json]: What are you using React-com for?

File: client/src/index.jsx [hunk 9]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
+// import CommentSection from "./components/CommentSection.jsx";
 import Comments from "./components/Comments.jsx";
 
 ReactDOM.render(<Comments />, document.getElementById("reviews"));
```
Comment (blbst123) [inline client/src/index.jsx]: Why is this component called `Comments` but the `div` is called reviews?

File: server/index.js [hunk 10]
```diff
@@ -1,9 +1,29 @@
...
+const db = require("../database/index.js");
+// const cors = require("cors");
 let app = express();
 
+// app.use(cors());
+app.use(express.static(__dirname + "/../client/dist/"));
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.get("/review_id/", (req, res) => {
+  const restaurantId = req.params.id;
```
Comment (blbst123) [inline server/index.js]: When you get reviews, why is the path called 'review_id' when then id you are specifying is for the restaurant?

File: server/index.js [hunk 11]
```diff
@@ -1,9 +1,29 @@
...
+
+  db.getComments(restaurantId, (err, results) => {
+    console.log(err, results);
+    if (err) {
+      console.log("err");
+    } else {
+      console.log(results);
```
Comment (blbst123) [inline server/index.js]: Might not be necessary to console.log the result here

File: server/index.js [hunk 12]
```diff
@@ -1,9 +1,29 @@
...
+      res.status(200).send(results);
+      console.log("successful GETs");
+    }
+  });
+});
 let port = 3001;
 
 app.listen(port, function() {
-  console.log(`listening on port ${port}`);
+  console.log(`listening on ports ${port}`);
```
Comment (blbst123) [inline server/index.js]: You are only listening on a single port right. Why ports

File: client/src/components/User.jsx [hunk 13]
```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import OnHover from "./OnHover.jsx";
+
+class User extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
```
Comment (blbst123) [inline client/src/components/User.jsx]: This is a really large component. Have you considered breaking it up into more smaller components? Will make it easier to read and debug/test.

File: client/src/components/OnHover.jsx [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div border="1px solid width:50px">
+        <ul className="onHov">
+          <li>
+            <svg
+              className="shareReviewIcon"
```
Comment (blbst123) [inline client/src/components/OnHover.jsx]: Write these in your CSS file. Will make this component a lot smaller and easier to separate the HTML from the CSS

File: client/src/components/HelpfulBtn.jsx [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+      buttonFunny: true,
+      buttonCool: true,
+      show: false,
+      click: 1
+    };
+    this.handleClick1 = this.handleClick1.bind(this);
+    this.handleClick2 = this.handleClick2.bind(this);
+    this.handleClick3 = this.handleClick3.bind(this);
+  }
+  handleClick1() {
```
Comment (blbst123) [inline client/src/components/HelpfulBtn.jsx]: Consider better naming conventions other than 1,2,3

File: client/src/components/CommentsUser.jsx [hunk 16]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+class CommentsUser extends React.Component {
```
Comment (blbst123) [inline client/src/components/CommentsUser.jsx]: Refactor to export default at this point.
```suggestion
export default class CommentsUser extends React.Component {
```

File: client/src/components/CommentsUser.jsx [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+          />
+        </span>
+        <span className="dateWritten">{this.props.data.date}</span>
+        {/* comments */}
+        <p className="userText">{this.props.data.comments}</p>
+      </div>
+    );
+  }
+}
+export default CommentsUser;
```
Comment (blbst123) [inline client/src/components/CommentsUser.jsx]: Don't need this if you export on top.

File: package.json (hunk 7 above)
Comment (iamasianman) [reply package.json]: @ bill it helps to have a live test to see if it works right away and a video says they highly recommend that you should watch it always

File: package.json (hunk 8 above)
Comment (iamasianman) [reply package.json]: @bill i accidentally miss spelled react dom but ill uninstall it

File: server/index.js (hunk 10 above)
Comment (iamasianman) [reply server/index.js]: @ bill i will make the changes so that my get request to make it sound better

File: client/src/components/HelpfulBtn.jsx (hunk 15 above)
Comment (iamasianman) [reply client/src/components/HelpfulBtn.jsx]: @blbst123 okay

Comment (iamasianman) [reply client/src/components/HelpfulBtn.jsx]: @blbst123 okay done

File: client/src/components/OnHover.jsx (hunk 14 above)
Comment (iamasianman) [reply client/src/components/OnHover.jsx]: @blbst123 resolved
